### DISCUSSION TITLE
Select certain fields from a collection without using View collection?

### DISCUSSION THREAD

**Discussion by jellypeep** [pocketbase/pocketbase#2064] [Q&A] [answered]
Is it possible to do something like this:

`const result = await pb.collection("users").getFullList({ fields: ["firstName", "lastName"]})`

Being able to write custom SQL queries through View collection is nice but it would have been nice if I could do something like the above.

**Comment by ganigeorgiev** [comment, marked as answer]
This was requested several times already (_you can also check #1880, #354, #259, etc._).

It is in the [roadmap](https://github.com/orgs/pocketbase/projects/2?pane=issue&itemId=16153865) and eventually will be implemented sometime in the future but for now is a low priority.
